web-module/src/main/webapp/resources/structEditJS/StructEditor.js

```diff
@@ -642,11 +642,23 @@
       mceFields.each(function(formfield) {
         if ((typeof tinyMCE !== 'undefined') && tinyMCE.get(formfield.id)) {
           _me._updateOneTinyMCETextArea(tinyMCE.get(formfield.id));
+        } else if (typeof tinyMCE !== 'undefined') {
+          console.debug('_updateTinyMCETextAreas: add delayed update ');
+          $$('body')[0].observe('celRTE:finishedInit', _me._finishLoadingTinymMce.bind(_me));
+        } else {
+          console.debug('_updateTinyMCETextAreas: skip no tinyMCE');
         }
       });
       console.log('_updateTinyMCETextAreas: end ', formId);
     },
 
+    _finishLoadingTinymMce : function(event) {
+      const _me = this;
```

Comment:
I don't see why local scoped variables should have a `_` prefix.
Perhaps `me` can even be inlined in this case (since we're not dealing with inner functions here).

Reply:
yes, that is true. But we have done this for years and this is a bug fix not a refactoring. I suggest that we keep the notion inside of one js-file the same. This js-file needs ES6 refactorings like many js files in the project.